tools/nhlt-parse.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright(c) 2020 Intel Corporation. All rights reserved.
+#
+# usage: nhlt-parse.py <nhtl-dump>
+#
+# examples:
+#   cat /sys/firmware/acpi/tables/NHLT >nhtl.bin
+#   ./nhtl-parse.py nhtl.bin
```

Comment:
can we directly read the /sys/firmware/acpi/tables/NHLT here?

Reply:
python can use absolute paths, this is fine